OpenRA.Mods.Common/Traits/SupportPowers/NukePower.cs

```diff
@@ -197,5 +218,72 @@ public void Activate(Actor self, WPos targetPosition)
 				});
 			}
 		}
+
+		public override void SelectTarget(Actor self, string order, SupportPowerManager manager)
+		{
+			Game.Sound.PlayToPlayer(SoundType.UI, manager.Self.Owner, Info.SelectTargetSound);
+			self.World.OrderGenerator = new SelectNukePowerTarget(self, order, manager, info, MouseButton.Left);
+		}
+	}
+
+	public class SelectNukePowerTarget : OrderGenerator
```

Comment:
This is 99% duplicating `SelectGenericPowerTarget` so please subclass that instead and override `RenderAnnotations`.